assets/src/edit-story/elements/video/karma/elementMinSizeAndPlayback.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { useInsertElement } from '../../../components/canvas';
+
+describe('Element min size and playback', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  it('should render ok', () => {
```

Comment:
This could probably be removed? Looks like it exists already elsewhere.

Reply:
I think it can go both ways:

Should be removed:
- It's unnecessary in general because either way the next expect will fail if it's not rendered correctly.

Should not be removed:
- You want to fail early with an accurate message.
- In the case of a focused tests, the other "should render ok" might not be fired.
- Or fired in a different order?

Reply:
Hmm, if this fails then probably all the tests everywhere are going to fail anyway, elsewhere as well, not sure if it makes sense to add this to each file separately. No strong opinion though, if you have a preference towards keeping it then fine by me.

It would be just good to make it consistent throughout the tests then.

Reply:
I would say it's not that beneficial in one place if you cannot guarantee the order of tests (test files).
@dvoytenko @barklund ?

Reply:
I don't think we need these basic tests in every test file. If it's a file testing X, doing a basic test for just the presence of X not failing is fine, but this doesn't appear to be that.